src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -173,6 +173,21 @@ public Category GetCategory
         }
 
         /// <summary>
+        /// Get Element Type
+        /// </summary>
+        public Element ElementType
+        {
+            get
+            {
+                // Get the active Document
+                Autodesk.Revit.DB.Document document = DocumentManager.Instance.CurrentDBDocument;
+
+                return document.GetElement(this.InternalElement.GetTypeId()).ToDSType(true);
```

Comment:
According to the Revit API docs, `GetTypeId()` returns:
>The id of the element's type, or invalid element id if the element cannot have type assigned. 

That means that `GetElement(...)` will fail if an invalid element id is returned. Can we add a check first like:
```c#
var typeId = this.InternalElement.GetTypeId();
if(typeId = ElementId.InvalidElementId){
  return null;
}else{
  return document.GetElement(typeId).toDSType(true);
}
```